src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -65,10 +67,14 @@ export const ArtworkSidebarFragmentContainer = createFragmentContainer(
       sale_artwork {
         lot_label
       }
+      sale {
+        is_closed
+      }
```

Comment:
This doesn’t get used inside this file, so it shouldn’t be fetching the data here.